### DISCUSSION TITLE
Announcing Sionna v0.11

### DISCUSSION THREAD

**Discussion by SebastianCa** [NVlabs/sionna#51] [Announcements]
We are happy to announce the Sionna v0.11 release:

# Features
- Support for optical fiber simulations
     * Split step Fourier method (SSFM)
     * Erbium doped fiber amplifier
     * New tutorial notebook on optical fiber simulations
     * Refactorization of the channel documentation
- BCJR decoder for convolutional codes
- Support for Turbo codes
     * Turbo encoding layer
     * Turbo decoding layer
     * New interleaver layer supporting the LTE Turbo code interleaver patterns (cf. 36.212)
     * Utility functions for code generation and for puncturing
- Improved scrambling layer
     * Support for custom scrambling sequences
     * Utility function to support scrambling sequences as defined in for 38.211
- Adds LDPC rate-matching output interleaver to LDPC5GEncoder and LDPC5GDecoder
- Utility function to generate random regular LDPC parity-check matrices
- Zero-forcing (ZF) and matched filter (MF) MIMO equalizers
- Tensor utility function for pseudo inverse computation
- Several MIMO utility functions for channel whitening and complex-to-real valued representations
- (OFDM) MIMO maximum likelihood detector with prior
- Utility layer for computing logits on constellation points from LLRs
- New tutorial on using the DeepMIMO dataset with Sionna
- Restructured website
     * Made with Sionna section
     * Direct links to Github discussion/issues
 
# Breaking Changes
- `keep_state` in LDPCBPDecoder and LDPC5GDecoder has been replaced by `stateful` flag and the decoder now returns the states instead of keeping an internal reference
- TDL and CDL power delay profiles are now normalized to have unit total power (related to issue #12).
 
# Fixes
- (#38) Fixes issue with PanelArray()
- (#43) Fixes issue with MLDetector
- Adds support for TensorFlow 2.10
- Bug fix in SymbolLogits2Moments
- Bug fix in mapping module related to double precision
- Adds missing reference of generate_dense_polar in sionna.fec.polar.__init__
- Improves stability of graph mode in Polar SCL decoding
- Issue of LDPC decoding when graph is not pruned
- Fixes missing attribute in Linear Encoder
- Improved selection of polynomials for convolutional codes
- Removes deprecated items in pylintrc due to updated version of Lynter
- Typos in the documentation
- Corrects the LoS path power calculation oft the 3GPP 38.901 channel models
- Fixes a bug in the effective noise variance computation of the lmmse_equalizer
